backend/migrations/versions/458572580709_remove_dataset_table_read_permissions_from_env_admins.py

```diff
@@ -30,6 +35,53 @@ def get_session():
     return session
 
 
+Base = declarative_base()
+
+
+class Dataset(Resource, Base):
```

Comment:
Do we need all of the below columns / do they even exist on base `dataset` table anymore?

Can we just query `dataset` table for 
`environmentUri` 
`SamlAdminGroupName`
`stewards`
`datasetUri`

Reply:
No we don't need them all, I just added the Dataset that was defined at the time this migration script was added, instead of importing the Dataset model that might be newer. This is the table that exists at this point of the migration. Although we should never go and modify alembic scripts (we create new ones), having the Dataser model prevents users to add things on columns that don't exist at this stage